lib/routers/profile/pil.js

```diff
@@ -153,14 +153,20 @@ router.put('/:pilId/revoke',
 );
 
 router.put('/:pilId/transfer',
+  permissions('pil.transfer'),
```

Comment:
This permission does not currently exist, so merging will effectively turn off PIL transfers.